.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# nf-core/methylseq: Contributing Guidelines
+
+Hi there! Many thanks for taking an interest in improving nf-core/methylseq.
+
+We try to manage the required tasks for nf-core/methylseq using GitHub issues, you probably came to this page when creating one. Please use the pre-filled template to save time.
+
+However, don't be put off by this template - other more general issues and suggestions are welcome! Contributions to the code are even more welcome ;)
+
+> If you need help using or modifying nf-core/methylseq then the best place to ask is on the pipeline channel on [Slack](https://nf-co.re/join/slack/).
```

Comment:
Shouldn't we list the actual [nf-core/methylseq pipeline channel](https://nfcore.slack.com/channels/methylseq) instead?